### ISSUE TITLE
Eclipse in single-host mode causes SSL error

### ISSUE THREAD

**Issue by PirateBread** [eclipse-che/che#18142]
### Describe the bug
Eclipse Che deployed in Single-Host mode with a valid internal SSL certificate is throwing a self-signed SSL error.

### Che version
 - [ ] latest
### Steps to reproduce
Deploy Eclipse with Helm Chart with exposureStrategy set to "single-host" and exposureType "native"
Deploy custom SSL certificate and update "CHE_INFRA_KUBERNETES_TLS__SECRET" config map value.

### Expected behavior
No SSL error for clients, and can browse dashboard.

### Runtime
   - [ ] kubernetes - 1.15.12-gke.20

### Screenshots
![image](https://user-images.githubusercontent.com/6953561/96572626-fb10bc00-12c4-11eb-88c7-a426757414d3.png)

### Installation method
   - [ ] helm

### Environment
   - [ ] Cloud
       - [ ] GCE

### Eclipse Che Logs
tbc

**Comment by PirateBread** [comment]
Update: The issue is related to access to KeyCloak, the error about the SSL does not seem to be related to the certificate but rather access to KeyCloak:

![image](https://user-images.githubusercontent.com/6953561/96578485-2d261c00-12cd-11eb-8028-6a55cbecd402.png)

**Comment by sleshchenko** [comment]
@PirateBread It would be useful if you provide the exact steps you performed to deploy and configure Che.

**Comment by sparkoo** [comment]
@PirateBread did you restart the keycloak and che pods after updating the tls secret?

**Comment by PirateBread** [comment]
Thank you @sleshchenko - I resolved the issue.... the problem is that the client could not access the KeyCloak URL and is not related to the SSL certs being untrusted.

The client javascript was trying to access : http://keycloak:5050 and failing - causing it to throw this generic "self-signed SSL" error.

Once I updated the KeyCloak URL to point to the KeyCloak Ingress - the self-signed SSL error went away.
